compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposePanel.desktop.kt

```diff
@@ -213,13 +214,13 @@ class ComposePanel @ExperimentalComposeUiApi constructor(
             _composeContainer?.windowContainer = value
         }
 
-    override fun add(component: Component): Component {
-        return super.add(component)
-    }
-
-    override fun remove(component: Component) {
-        super.remove(component)
-    }
```

Comment:
You cannot remove it - it breaks binary compatibility

Reply:
Put it back and added a comment.